x-pack/plugins/synthetics/server/synthetics_service/get_service_locations.ts

```diff
@@ -41,13 +43,22 @@ export async function getServiceLocations(server: UptimeServerSetup) {
       locations: Record<string, ManifestLocation>;
     }>(server.config.service!.manifestUrl!);
 
-    Object.entries(data.locations).forEach(([locationId, location]) => {
+    const isProd = process.env.NODE_ENV === 'production';
```

Comment:
i will recommend using flag that is coming from plugin context to determine if the evn is dev or prod

Reply:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/synthetics/public/plugin.ts#L213